src/metabase/api/alert.clj

```diff
@@ -20,12 +20,20 @@
 
 (api/defendpoint GET "/"
   "Fetch all alerts"
-  [archived]
-  {archived (s/maybe su/BooleanString)}
-  (as-> (pulse/retrieve-alerts {:archived? (Boolean/parseBoolean archived)}) <>
+  [archived user_id]
+  {archived (s/maybe su/BooleanString)
+   user_id  (s/maybe su/IntGreaterThanZero)}
```

Comment:
It would be nice to update the docstring to clarify what `user_id` does if it's passed. It's not immediately clear if `user_id` returns ones created by that user, or ones that user is subscribed to, or ones that User can see/run/etc